### PR TITLE
Creating result element / price input field component

### PR DESCRIPTION
**Pull request by l-goede**
Build result element component and style with css 
https://sagit21-resultelement-gzxhqcl2.herokuapp.com/storybook/?path=/story/component-footer--nav-bar

### CODE REVIEW

File: src/app/components/Icons/NotificationIcon.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ function NotificationIcon(): JSX.Element {
   return (
     <svg
       xmlns="http://www.w3.org/2000/svg"
```
Comment (jujuzitroni) [inline src/app/components/Icons/NotificationIcon.tsx]:
```suggestion
    
```

File: src/app/components/Icons/Logo.tsx [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+      >
+        <g id="Gruppe_313" data-name="Gruppe 313" transform="translate(3.344)">
+          <line
+            id="Linie_34"
+            data-name="Linie 34"
+            x1="1.216"
+            y2="188.485"
+            transform="translate(5.472)"
+            fill="none"
+            stroke="#866835"
```
Comment (J-Bannach) [inline src/app/components/Icons/Logo.tsx]: Maybe use CSS Variables instead?

**Review by jujuzitroni — COMMENTED** [review]
lgtm 🚀

**Review by Maurizio-F — APPROVED** [review]
lgtm! 💯

**Review by J-Bannach — APPROVED** [review]
lgtm 👍

File: src/app/components/Icons/Logo.tsx [hunk 3]
```diff
@@ -0,0 +1,320 @@
+import React from 'react';
+
+function Logo(): JSX.Element {
+  return (
+    <svg width="237.626" height="216.05" viewBox="0 0 237.626 216.05">
+      <g
+        id="Gruppe_345"
+        data-name="Gruppe 345"
```
Comment (lajanzen) [inline src/app/components/Icons/Logo.tsx:8]: You can erase the ids and the data names.

**Review by lajanzen — COMMENTED** [review]
🚀

**Pull request merged**